### PR TITLE
view log pagination

### PR DESCRIPTION
**Pull request by bnchdan**
Hello,

I have added pagination to the view log page.

The request is made with the go2page function.

image

### CODE REVIEW

File: public/viewlog.php [hunk 1]
```diff
@@ -44,11 +44,40 @@
...
+				break;
+			}
+		}
+	}
+	
+	if ($flag_fDomain == 0 ){
+		die('Unknown domain');
+	}	
+
+	$page_number_aux=escape_string($_GET['page']);
```
Comment (DavidGoodwin) [inline public/viewlog.php:66]: I'd not use 'escape_string' here - i think you'd be best using e.g.

$page_number_aux = (int) ($_GET['page'] ?? 0);

(in more english like ... that's : give me the 'page' url parameter, and if it doesn't exist default it to zero; and cast the entire thing to a number)

**Comment by DavidGoodwin** [comment]
Thanks for taking the time to contribute :) 

Aside from your formatting not being quite right, I think it looks good.

File: public/viewlog.php (hunk 1 above)
Comment (bnchdan) [reply public/viewlog.php]: I modified the line

**Comment by bnchdan** [comment]
Thanks!

**Pull request merged**